package.json

```diff
@@ -226,7 +226,10 @@
     "lint:other": "yarn prettier --list-different && yarn stylelint",
     "lint:code": "eslint -c .eslintrc.json --ext .js,.ts,.tsx .",
     "lint:spelling": "cspell -c .cspell.json 'src/script/**/*.{js,ts,tsx,md}' src/i18n/en-US.json",
-    "postinstall": "yarn configure && cd server && yarn  && cd .. && husky install",
+    "check:circular-dependencies": "yarn check:circular-dependencies:app && yarn check:circular-dependencies:auth",
+    "check:circular-dependencies:app": "dpdm -T ./src/script/main/app.ts",
+    "check:circular-dependencies:auth": "dpdm -T ./src/script/auth/main.tsx",
```

Comment:
we could go for `circular-deps` maybe. But honnestly, since this is run automatically, I don't think the name is a big deal

Reply:
I am a fan of speaking variable and function names. This is the clearest it can get.
Usually, we don't have to run this command by hand. And if we do, we can copy paste it into the console I think :)
I would prefer leaving it this way